meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -1,7 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 BRANCH_lmp = "master"
-SRCREV_lmp = "d132ef7456a06697b782e0c5ab6260e0a89e512e"
+SRCREV_lmp = "cfa32fc4658c67d7ca0b45827cb64e2aeea729b3"
```

Comment:
Rev should be 653a32d6eab22168b52d386c195216f11561b20b no? By looking at https://github.com/foundriesio/aktualizr-lite/commit/653a32d6eab22168b52d386c195216f11561b20b.

Reply:
yep, I screwed up, fixed & updated PR, thx.